tasks/tangd-custom-port.yml

```diff
@@ -0,0 +1,61 @@
+---
+# This task tells SELinux for the port that we want the tangd service to use when distribution Fedora
+- name: Allow the custom port for tangd_port_t in SELinux
+  import_role:
+    name: fedora.linux-system-roles.selinux
+  vars:
+    selinux_ports: 
+      - ports: "{{ nbde_server_port }}"
+        proto: tcp
+        setype: tangd_port_t
+        state: present
+
+# This block creates the override file for systemd with the new port that we have requested
+- name: Create override file
+  block:
+    # This task checks if the directory /etc/systemd/system/tangd.socket.d exist and registers the value in systemd__system__tangd_socket
+    - name: Check if directory /etc/systemd/system/tangd.socket.d exist
+      stat:
+        path: /etc/systemd/system/tangd.socket.d
+      register: systemd_system_tangd_socket
+
+    # This tasks Create the /etc/systemd/system/tangd.socket.d directory if it does not exist
+    - name: Create a directory if it does not exist
+      file:
+        path: /etc/systemd/system/tangd.socket.d
+        state: directory
+        mode: '0755'
+      when: systemd_system_tangd_socket.stat.exists
+
+    # This tasks creates the file with the port entry that we want tangd to listen to 
+    - name: Creates the file with the port entry that we want tangd to listen to 
```

Comment:
Why use a `blockinfile` here instead of `template`?
We'll also need a proper `ansible_managed` header here - many of our users require a custom multi-line value for `ansible_managed` - so our roles have standardized on using
`{{ ansible_managed | comment }}` in files managed by the roles. If you use `template` it will be easy - if you use `blockinfile` then you'll need to have some additional code in order to ensure a proper ansible managed header.